pom.xml

```
@@ -101,7 +101,7 @@
     <jetty11.version>11.0.11</jetty11.version>
     <jakarta.servlet>5.0.0</jakarta.servlet>
 
-    <kafka.version>3.2.1</kafka.version>
+    <kafka.version>3.4.0</kafka.version>
```

CVE fixed in 3.2.3 https://downloads.apache.org/kafka/3.2.3/RELEASE_NOTES.html, 3.4 would require upgrade:

```suggestion
    <kafka.version>3.2.3</kafka.version>
```